Travis errors:
```
muskathlon/forms/order_material_form.py:152:21: E122 continuation line missing indentation or outdented
muskathlon/forms/order_material_form.py:153:17: E122 continuation line missing indentation or outdented
muskathlon/forms/order_material_form.py:183:5: E303 too many blank lines (2)
```

--- a/muskathlon/forms/order_material_form.py
+++ b/muskathlon/forms/order_material_form.py
@@ -21,7 +21,6 @@ try:
 except ImportError:
     _logger.warning('Please install magic in order to use Muskathlon module')
 
-
 if not testing:
     # prevent these forms to be registered when running tests
 
@@ -146,12 +145,11 @@ if not testing:
             picture = extra_values['large_picture']
             email_template = self.env.ref(
                 'muskathlon.order_material_mail_template2')
-            email_template.send_mail(self.main_object.id,
-                                     raise_exception=False,
-                                     force_send=True, email_values={
-                    'attachments': [('picture.jpg', picture)]
-                })
+            email_template.send_mail(
+                self.main_object.id, raise_exception=False, force_send=True,
+                email_values={'attachments': [('picture.jpg', picture)]})
             return True
+
 
     class OrderMaterialFormFlyer(models.AbstractModel):
         _name = 'cms.form.order.material'

--- a/muskathlon/forms/order_material_form.py
+++ b/muskathlon/forms/order_material_form.py
@@ -150,7 +150,6 @@ if not testing:
                 email_values={'attachments': [('picture.jpg', picture)]})
             return True
 
-
     class OrderMaterialFormFlyer(models.AbstractModel):
         _name = 'cms.form.order.material'
         _inherit = 'cms.form.order.material.mixin'
@@ -176,7 +175,6 @@ if not testing:
                 'datas_fname': filename,
                 'name': filename
             })
-
 
     class OrderMaterialFormChildpack(models.AbstractModel):
         _name = 'cms.form.order.muskathlon.childpack'

